test/classes/claridge/baseConstructorCall.chpl

```diff
@@ -1,17 +1,18 @@
 class BaseClass {
   var a, b: int;
 
-  proc BaseClass(b: int) {
+  proc init(b: int) {
     this.b = b;
+    super.init();
```

Comment:
@bradcray - I've been taking the policy of putting things that are obviously Phase 1 explicitly before a super.init().  There's a couple of more complicated tests (that didn't work for other reasons) but that naturally seemed to have the Phase 1/Phase 2 divide, so I tried inserting one midway through those (for instance, https://github.com/chapel-lang/chapel/blob/master/test/studies/madness/aniruddha/madchap/MRA.chpl#L62).  Should I be taking the policy of "Phase 2 as much as possible" for those as well?

I was going to avoid rearranging to make a clear Phase 1/Phase 2 divide as much as possible, but imo favoring the "easy" approach for Phase 1 only bodies just means less opportunities to iron the errors out of the more complicated portion, which was part of the intention behind switching the constructors over now rather than focusing on default initializers/noinit.

I was just thinking a list of initializers that could trivially be Phase-1-only would be a good idea, and I think such a list would be just as easily populated from "ugh, why did I have to add this super.init()" moments as "oh, I guess that could be Phase 1 only" (and might even be more quickly noticed in the former).

I'm not really arguing that this is a policy anyone that helps me should follow (since that seems annoying), but I think it is good for *me* to do it, at least.  Thoughts?